home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>YoutTube - Custome</title>
+</head>
+<body>
+    <ul class="header">
+        <li ><a href="#">YoutTube</a></li>
+        <li style="float: right;"><a href="#">LogOut</a></li>
+    </ul>
+    <div class="side-navigation">
+        <ul class="side-nav">
+            <li><a href="#" style="font-weight: bold">Subscription Channels</a></li>
+            <li><a href="#">Channel-1</a></li>
+            <li><a href="#">Channel-2</a></li>
+            <li><a href="#">Channel-3</a></li>
+        </ul>
+        <section>
```

Comment:
- [ ] Kindly please use `heading` after `section ` tag

Reply:
- [ ] Kindly I recommend you to use `headings` tag after `section ` tag. Sections should always to have _`headings`_ like `h1, h2, h3` which describes what the section is about